### PR TITLE
[NG] FAQ Page

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR creates FAQ page under company. 
https://deploy-preview-1319--layer5ng.netlify.app/company/faq

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
We are missing a magnifying glass image in the search bar.

**Review by Tanuj22 — COMMENTED** [review]
Also you can get the search working. A simple javascript search is needed for this instead of something like `lunarJS`

File: src/sections/Faq/index.js [hunk 1]
```diff
@@ -14,11 +14,7 @@ import {
...
 } from "../../reusecore/Accordion";
 
-import { IoIosArrowDown } from "react-icons/io";
-import { FaPlus } from "react-icons/fa";
-
-import particle1 from "../../assets/images/app/particle/07.png";
-import particle2 from "../../assets/images/app/particle/08.png";
+import { IoIosArrowDown, IoIosArrowUp } from "react-icons/io";
 
 import data from "../../assets/data/faq";
```
Comment (Tanuj22) [inline src/sections/Faq/index.js:19]: Assuming that this faq would be different for company from meshery. It should be a separate file.

**Comment by Jashpatel1** [comment]
> Also you can get the search working. A simple javascript search is needed for this instead of something like `lunarJS`

Can anyone explain what kind of search is required @Tanuj22 @Nikhil-Ladha @leecalcote? A simple array search?

**Comment by Jashpatel1** [comment]
> We are missing a magnifying glass image in the search bar.

The unicode doesn't work. I couldn't find any other method apart from manually fixing its position.

**Comment by Tanuj22** [comment]
> > Also you can get the search working. A simple javascript search is needed for this instead of something like `lunarJS`
> 
> Can anyone explain what kind of search is required @Tanuj22 @Nikhil-Ladha @leecalcote? A simple array search?

Yeah should be enough.

File: src/sections/Faq/index.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/Faq/index.js:19]: What does this datafile look like? Will each FAQ be written to an mdx (or other type) file?

**Comment by leecalcote** [comment]
@Tanuj22 we'll want for search to include the faq data files.

**Comment by leecalcote** [comment]
> @Tanuj22 we'll want for search to include the faq data files.

@Tanuj22 I'm just now seeing the comments asking "what type of search" should be here. It looks like you're already accounting for the need to be able to search through FAQs. This is great. Long-term, the more FAQs that are build, the more that we will want them included in a site-wide search. For now, arrays will do the trick like you said.

File: src/sections/Faq/index.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/Faq/index.js:19]: Its an array

**Review by leecalcote — DISMISSED** [review]
I think we're good to merge with a follow up on search.

File: src/sections/Faq/index.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/Faq/index.js:19]: @Jashpatel1 Change this. And what's the update on search?

**Comment by leecalcote** [comment]
I will push some faqs to this PR branch.

**Pull request merged**